docker-compose.yaml

```diff
@@ -2,17 +2,17 @@ version: '3'
 
 services:
     esserver:
-        image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.4
+        image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.6.0
         volumes:
             - "./esdata:/usr/share/elasticsearch/data/"
         environment:
             - discovery.type=single-node
             - node.name=esserver
             - network.host=_site_
-            - "ES_JAVA_OPTS=-Xms4g -Xmx8g"
+            - "ES_JAVA_OPTS=-Xms4g -Xmx20g"
```

Comment:
This value is hard-coded because this will only be used on the ELK server? Otherwise the 20g max seems high if others will use it

Reply:
Previously I had put these values in a .env file, but it didn't seem to get picked up there. I have no expectation that anyone else (outside the Reactome team) will use this on any other server. I'll probably try to move that setting to a .env file again and configure it differently, but it's not a top priority.